undertow/src/main/java/org/wildfly/extension/undertow/ServerAdd.java

```diff
@@ -38,16 +38,16 @@
 /**
  * @author <a href="mailto:tomaz.cerar@redhat.com">Tomaz Cerar</a> (c) 2013 Red Hat Inc.
  */
-class ServerAdd extends AbstractBoottimeAddStepHandler {
+class ServerAdd extends AbstractAddStepHandler {
```

Comment:
This really simplifies / speeds up our testing if noting else.
so you can add servers / hosts without need for reload.

But in general I think it is a good thing to have.